src/stores/user-store.js

```diff
@@ -34,11 +34,21 @@ class UserStore extends EventEmitter {
         this._users.push(user);
     }
 
+    _updateUser (user) {
+        const id               = user.id;
+        this._users[id - 1]    = user;
```

Comment:
We use type conversion here. https://www.w3schools.com/js/js_type_conversion.asp
IMO it's better to use some method to find an element in array with exact `id`. Because it'll give  us more stable solution